src/components/postTags/PostTagList.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useEffect, useState } from "react"
+import { TagContext } from "../tags/TagProvider"
+import { PostTagContext } from "./PostTagProvider"
+import Badge from 'react-bootstrap/Badge'
+import "./PostTagForm.css"
+
+export const PostTagList = ({postId}) => {
+    const { tags, getTags } = useContext(TagContext)
+    const { getPostTagsByPostId } = useContext(PostTagContext)
+    const [ thisPostTags, setThisPostTags ] = useState([]) 
+		const [ selectedPostTags, setSelectedPostTags ] = useState([]) 
```

Comment:
Do we need this state variable at all in this component? It seems like it's not being used and we could maybe just get rid of it?

Reply:
Specifically, the `selectedPostTags` state variable

Reply:
Good Catch!